Core/Src/sensors/steering.c

```diff
@@ -50,31 +52,10 @@ void steering_read(void* argument) {
             }
         }
 
-        // The function converts the value of the ADC to the value in degrees of the
-        // screeching of the steering wheel, Also informs the steering wheel direction.
-        // SPAN_ALINHAMENTO is just a span to still consider the steering wheel in the
-        // center up to a certain amount
-        if (current_read > ALIGNED_STEERING + SPAN_ALIGNMENT) {
-            set_global_var_value(INTERNAL_WHEEL, LEFT);
-            set_global_var_value(STEERING_WHEEL, calculate_steering(current_read)
-                                                     - ALIGNED_STEERING_ANGLE);
-        } else if (current_read < ALIGNED_STEERING - SPAN_ALIGNMENT) {
-            set_global_var_value(INTERNAL_WHEEL, RIGHT);
-            set_global_var_value(STEERING_WHEEL, (ALIGNED_STEERING_ANGLE
-                                                  - calculate_steering(current_read)));
-        } else {
-            set_global_var_value(INTERNAL_WHEEL, CENTER);
-            set_global_var_value(
-                STEERING_WHEEL,
-                (uint16_t)(calculate_steering(current_read) - ALIGNED_STEERING_ANGLE));
-        }
-
-        STEERING_WHEEL_t steering_wheel_datalog = get_global_var_value(STEERING_WHEEL);
-        INTERNAL_WHEEL_t internal_wheel_datalog = get_global_var_value(INTERNAL_WHEEL);
-
-        log_data(ID_STEERING_WHEEL, steering_wheel_datalog);
-        log_data(ID_INTERNAL_WHEEL, internal_wheel_datalog)
+        STEERING_WHEEL_t steering_wheel = (int16_t)calculate_steering(current_read);
```

Comment:
```suggestion
        STEERING_WHEEL_t steering_wheel = calculate_steering(current_read);
```
Esse cast não muda nada, e com o outro comentário ele ja vai receber um int16_t msm